docs/UserGuide.adoc

```diff
@@ -17,10 +17,12 @@ By: `Team W13-B1`      Since: `Mar 2018`      Licence: `MIT`
 
 == Introduction
 
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+ContactSails is for those users who *prefer to use a desktop app for managing contacts*. More importantly, it is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ContactSails can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to <<Quick Start>> to get started. Enjoy!
```

Comment:
Just a minor note, I think we can remove the "those" between for and users in the first line.
Aside from that, it looks alright!

Reply:
Okay, will update it accordingly